packages/crypto.base32-hash/src/index.ts

```diff
@@ -2,8 +2,17 @@ import crypto from 'crypto'
 import fs from 'fs'
 import { base32 } from 'rfc4648'
 
+const hash =
+  // @ts-expect-error -- crypto.hash is supported in Node 21.7.0+, 20.12.0+
+  crypto.hash ??
+  ((
+    algorithm: string,
+    data: crypto.BinaryLike,
+    outputEncoding: crypto.BinaryToTextEncoding
+  ) => crypto.createHash(algorithm).update(data).digest(outputEncoding))
+
 export function createBase32Hash (str: string): string {
-  return base32.stringify(crypto.createHash('md5').update(str).digest()).replace(/(=+)$/, '').toLowerCase()
+  return base32.stringify(hash('md5', str, 'buffer')).replace(/(=+)$/, '').toLowerCase()
```

Comment:
It seems this is because the default encoding of `digest` is `buffer`, and the third parameter of `crypto.hash` defaults to `hex`.